tests/integration/check_tests.rs

```diff
@@ -146,7 +146,7 @@ fn check_error_in_dep_then_fix() {
   let check_command = test_context.new_command().args_vec(["check", "main.ts"]);
 
   let output = check_command.run();
-  output.assert_matches_text("Check [WILDCARD]main.ts\nerror: TS234[WILDCARD]");
+  output.assert_matches_text("Check [WILDCARD]main.ts\nTS234[WILDCARD]");
```

Comment:
Errors don't all get dispalyed at once anymore--they get displayed to the screen after each group of roots gets type checked.